ui/tests/integration/components/secret-engine/configure-aws-test.js

```diff
@@ -222,107 +286,115 @@ module('Integration | Component | SecretEngine/ConfigureAws', function (hooks) {
         );
       });
 
-      // ISSUER FIELD TESTING
-      // the other tests where issuer is not passed do not show modals, so we only need to test when the modal should show up
-      test('it shows existing issuer, shows modal when saving changes, and does not call APIs on cancel', async function (assert) {
-        this.issuer = 'http://foo.bar';
-        this.server.post('/identity/oidc/config', () => {
-          assert.notOk(true, 'request should not be made to issuer config endpoint');
+      module('issuer field tests', function () {
+        // the other tests where issuer is not passed do not show modals, so we only need to test when the modal should shows up
```

Comment:
There was an explicit test and scenario where if the issuer config was not read, we should show a modal warning them that it could override it on save. I think that scenario has been refactored out of working

Reply:
Okay, I think I see where i went wrong there. I added in two more test that specifically check the modal message per the capabilities check of canRead. Let's chat tmrw however, if we want any failure on the identity/oidc/config endpoint to result in canRead === false.

Reply:
We didn't have a failure on read before because generally it doesn't matter, unless they update it we want to warn them

Reply:
Reverted back to previous behavior where any failure we show this specific warning message "if it exists".